recipes/tn93/meta.yaml

```diff
@@ -0,0 +1,18 @@
+package:
+  name: tn93
+  version: 1.0.2
+
+source:
+  git_rev: v1.0.2
+  git_url: https://github.com/veg/tn93.git
+
+requirements:
+  build:
+    - gcc
+    - cmake
+  run:
+    - libgcc
+
```

Comment:
Can you also please add a `about` section: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/bamtools/meta.yaml#L31